src/python/sdk/econia_sdk/utils/decimals.py

```diff
@@ -0,0 +1,106 @@
+import math
+from decimal import Decimal
+
+
+def get_lot_size_integer(smallest_decimal_unit: str, coin_decimals: int) -> int:
+    """
+    Given a decimal representation of the smallest possible unit and the
+    whole unit's decimals (e.g. ETH has 18 decimals), return the lot size.
+
+    Parameters:
+    * `smallest_decimal_unit`: Decimal (e.g. "0.001" for one thousandth) of
+      the smallest possible unit relative to a whole unit.
+    * `coin_decimals`: The number of decimals one whole unit of the coin
+      has (e.g. USDC has 6 decimals, ETH has 18).
+    """
+    smallest_decimal = Decimal(smallest_decimal_unit)
+    smallest_subunits = 10 ** (coin_decimals + math.log10(smallest_decimal))
```

Comment:
I also suggest the same for the decimals, e.g. `verify_int_input` because (un?)fortuantely Python isn't strictly typed